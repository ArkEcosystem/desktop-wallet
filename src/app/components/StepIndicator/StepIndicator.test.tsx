import React from "react";
import { render } from "@testing-library/react";

import { StepIndicator } from "./";

describe("StepIndicator", () => {
	it("should render", () => {
		const { container, asFragment } = render(<StepIndicator />);

		expect(container).toBeTruthy();
		expect(asFragment()).toMatchSnapshot();
	});
});
