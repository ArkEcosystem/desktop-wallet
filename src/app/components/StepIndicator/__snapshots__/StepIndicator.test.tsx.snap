// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepIndicator should render 1`] = `
<DocumentFragment>
  <ul
    class="StepIndicator__StepWrapper-sc-15t41is-1 iToWWb"
  >
    <li
      class="StepIndicator__StepStyled-sc-15t41is-0 dATCfl"
    />
    <li
      class="StepIndicator__StepStyled-sc-15t41is-0 khoCaB"
    />
  </ul>
</DocumentFragment>
`;
